.github/workflows/build-soroban-dev.yml

```diff
@@ -42,3 +44,32 @@ jobs:
           "network": ["standalone"],
           "options": ["", "--enable-horizon-captive-core", "--enable-soroban-rpc"],
         }
+
+  arm64:
+    uses: ./.github/workflows/build.yml
+    secrets:
+      DOCKERHUB_USERNAME: ${{ secrets.DOCKERHUB_USERNAME }}
+      DOCKERHUB_TOKEN: ${{ secrets.DOCKERHUB_TOKEN }}
+    with:
+      arch: arm64
+      tag: soroban-dev-arm64
+      core_ref: c0ad35aa19297e112d71fcc5755458495f99a237
```

Comment:
is it worthwhile to refactor the hardcoded core_ref since it's repeated now into a reference to ENV block:
```
env:
  CORE_REF: c0ad35aa19297e112d71fcc5755458495f99a237
```

Reply:
Unfortunately the `env` context variable is not available in the calls to other workflows, so if we define an env at the top of the file with that value, we can't use it here.